test/test_transforms.py

```diff
@@ -1078,7 +1078,7 @@ def test_rotate(self):
     def test_rotate_fill(self):
         img = F.to_pil_image(np.ones((100, 100, 3), dtype=np.uint8) * 255, "RGB")
 
-        modes = ("L", "RGB")
+        modes = ("L", "RGB", "F")
         nums_bands = [len(mode) for mode in modes]
         fill = 127
```

Comment:
Lets see if this works with `mode=="F"` and an int value `>1`.

Reply:
We can create `F` images with arbitrary float values in it, including > 1 it seems
```python
PIL.Image.fromarray(np.full((10, 10), 5.))
```